Formula/c/ca-certificates.rb

```diff
@@ -147,8 +147,14 @@ def linux_post_install
     rm(pkgetc/"cert.pem", force: true)
     pkgetc.mkpath
 
-    system_ca_certificates = Pathname.new("/etc/ssl/certs/ca-certificates.crt")
-    return if !system_ca_certificates.exist? || !system_ca_certificates.readable?
+    ca_certificate_paths = [
+      "/etc/ssl/certs/ca-certificates.crt", # Debian/Ubuntu, Alpine Linux, Arch Linux
+      "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem", # RHEL/CentOS/Fedora
+      "/etc/ssl/ca-bundle.pem", # SUSE/openSUSE
+    ]
+    system_ca_certificates = ca_certificate_paths.map { |p| Pathname.new(p) }
+                                                 .find { |pn| pn.file? && pn.readable? }
+    return unless system_ca_certificates
```

Comment:
For full generality, perhaps we could use the same set of patterns/heuristics that rustls's `openssl-probe` uses? I think that would be a nice future-proofing step here 🙂 

https://github.com/rustls/openssl-probe/blob/main/src/lib.rs